pkg/apis/agones/v1/gameserver.go

```diff
@@ -498,16 +551,6 @@ func ValidateResource(request resource.Quantity, limit resource.Quantity, resour
 	return validationErrors
 }
 
-// validateResources validate CPU and Memory resources
-func validateResources(container corev1.Container) []error {
```

Comment:
The validation of resources will be handled by `k8s_validation.ValidatePodTemplateSpec` method.